### PR TITLE
CSPL-887: Documentation review

### PR DESCRIPTION
**Pull request by eddikay**
ChangeLog.md - No changes. - Inbound links: Readme.md
Contributing.md - No changes. - Inbound links: None. Add from Readme.md
CustomResources.md - No changes.
OpenShift.md - No changes. It's a bit thin, but maybe there's no other detail required? - Inbound links: Readme.md
Faq.md - Remove. The only viable content remains is a quip about why HELM wasn't used.
README.md - Significant changes
Install.md - Cleanup
Images.md - Cleanup
StorageClass.md - Cleanup
Ingress.md - Adding numbers to procedures.
SmartStore.md - No changes.
PasswordManagement.md - No changes.
Examples.md -  No changes.
MultisiteExamples.md - No changes.

### CODE REVIEW

File: docs/Images.md [hunk 1]
```diff
@@ -51,47 +42,41 @@ docker tag splunk/splunk-operator:latest gcr.io/splunk-operator-testing/splunk-o
...
-On your Kubernetes workers, you can then import this image using:
+And on your Kubernetes workers, you can import the tarball using:
 
 ```
 docker load -i splunk-operator.tar.gz
 ```
 
 
-## Simple Script to Push Images
+## A Simple Script to Push Images
```
Comment (smohan-splunk) [inline docs/Images.md:65]: A "Sample" Script?

**Review by smohan-splunk — APPROVED** [review]

**Comment by vebken-splunk** [comment]
Seems like there are some merge conflicts to resolve, I'm assuming they are mostly relating to removal of Spark from the docs.

File: docs/Images.md [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
+Use the `RELATED_IMAGE_SPLUNK_ENTERPRISE` environment variable or the `image` custom resource parameter to change the location of your Splunk Enterprise image. Use the `RELATED_IMAGE_SPLUNK_SPARK` environment variable or the`sparkImage` parameter to change the location of the Spark image. 
+
+For additional detail, see the [Advanced Installation Instructions](Install.md) page, and the [Custom Resource Guide](CustomResources.md) page.
 
 
 ## Using a Private Registry
 
-If your Kubernetes workers have access to pull from a Private registry, it is
-easiest to retag and push the required images to directly to the private registry.
+If your Kubernetes workers have access to pull from a Private registry, it is easy to retag and push the required images to directly to the private registry.
```
Comment (vebken-splunk) [inline docs/Images.md:21]: nit: not related to your change specifically, but we have inconsistent capitalization between "Private registry" and "private registry"

File: docs/Images.md [hunk 3]
```diff
@@ -51,47 +42,41 @@ docker tag splunk/splunk-operator:latest gcr.io/splunk-operator-testing/splunk-o
...
 
 ```
 docker load -i splunk-operator.tar.gz
 ```
 
 
-## Simple Script to Push Images
+## A Simple Script to Push Images
+
+A sample script `build/push_images.sh`  is included with the opertor to push Docker images to multiple remote hosts using SSH. The script takes the name of a container and an image path, and pushes the image to all the entries in `push_targets`.
```
Comment (vebken-splunk) [inline docs/Images.md:67]: nit: operator is misspelled "opertor"

**Comment by vebken-splunk** [comment]
A general nitpick question I have reviewing the docs, when we are using "operator" as shorthand for "Splunk Operator" rather than the Kubernetes operator pattern, should it be capitalized?

File: docs/README.md [hunk 4]
```diff
@@ -40,96 +33,57 @@ and considered to be a "beta" quality release. We expect significant
 modifications will be made prior to its general availability, it is not
```
Comment (vebken-splunk) [inline docs/README.md:33]: Not related to your change, but we probably want to rewrite or remove this paragraph for GA.

File: docs/README.md [hunk 5]
```diff
@@ -40,96 +33,57 @@ and considered to be a "beta" quality release. We expect significant
...
 
-## Creating Splunk Enterprise Deployments
+1. Run the command to create a deployment named “s1”:
 
-The `Standalone` resource is used to create a single instance deployment
-of Splunk Enterprise. For example,  run the following command to create a
-deployment named “s1”:
 
 ```yaml
 cat <<EOF | kubectl apply -f -
```
Comment (vebken-splunk) [inline docs/README.md:89]: We'll want to update the apiVersion in the example to v1 per https://github.com/splunk/splunk-operator/pull/274

**Comment by vebken-splunk** [comment]
For all yaml examples, we should update the apiVersion to v1 per #274

File: docs/StorageClass.md [hunk 6]
```diff
@@ -65,84 +58,47 @@ spec:
...
-## Additional Providers
+## Additional Storage Providers
 
-The [introduction](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/)
-of Kubernetes’s [Container Storage Interface](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) (CSI)
-has made it easy for new vendors to offer innovative solutions for managing
-the persistence of containerized applications. Users of Kubernetes 1.13 or
-later are encouraged to take a look at
-[this list of CSI Drivers](https://kubernetes-csi.github.io/docs/drivers.html).
+The [introduction](https://kubernetes.io/blog/2018/01/introducing-container-storage-interface/) of Kubernetes’s [Container Storage Interface](https://kubernetes.io/blog/2019/01/15/container-storage-interface-ga/) (CSI) has made it easy for new vendors to offer innovative solutions for managing the persistence of containerized applications. Users of Kubernetes 1.13 or later are encouraged to take a review the Kubernetes [list of CSI Drivers](https://kubernetes-csi.github.io/docs/drivers.html).
```
Comment (vebken-splunk) [inline docs/StorageClass.md:95]: nit: "encouraged to take a review the Kubernetes" -> "encouraged to review the Kubernetes"

File: docs/Ingress.md [hunk 7]
```diff
@@ -461,7 +466,7 @@ Istio is a popular choice for its Service Mesh capabilities. However, Service Me
 
 It is important to confirm which Nginx Ingress controller you intended to implement, as they have *very* different annotations and configurations. For these examples, we are using the Kubernetes Ingress Nginx (option 1) and the Nginx Ingress Open Source (option 2).
 
-## Configuring Ingress Using Kubernetes Ingress Nginx <a name="Ingress-Nginx"></a>
+## Configuring Ingress Using Kubernetes Ingress Nginx 
 
 For instructions on how to install and configure the NGINX Ingress Controller, see the 
 [NGINX Ingress Controller GitHub repository](https://github.com/nginxinc/kubernetes-ingress/) and the [Installation Guide](https://kubernetes.github.io/ingress-nginx/deploy/).
```
Comment (vebken-splunk) [inline docs/Ingress.md:472]: This line needs to be modified https://github.com/nginxinc/kubernetes-ingress/ should be replaced with https://github.com/kubernetes/ingress-nginx

**Review by mgaldino-splunk — APPROVED** [review]

File: docs/Images.md (hunk 3 above)
Comment (vebken-splunk) [reply docs/Images.md:66]: This typo still seems to remain in the new commit.

**Review by vebken-splunk — CHANGES_REQUESTED** [review]

**Review by vebken-splunk — APPROVED** [review]

**Review by akondur — APPROVED** [review]

**Pull request merged**